Tests/testrunner.cpp

```diff
@@ -39,62 +39,70 @@
 /// Implementation of the googletest testrunner
 int main(int argc, char* argv[])
 {
+    std::string logLevel("all");
```

Comment:
TCLAP does not allow undefined parameters. gtest parameters then have to be passed after `--` which I don't like, e.g.:

```
./testrunner -l warn -- --gtest_filter=Foo
```